documentation/identifiers.md

```diff
@@ -76,67 +86,80 @@ There are many different kinds of organisations that give, receive or benefit fr
 
-Organisation Identifiers take the form:
+There are two parts to an organisation identifier:
 
->*Registration Agency Code* (dash) *Identifier*
+* **A list code** that describes the list the identifier is taken from
+* **An identifier** taken from that list.
 
-The Registration Agency Code is taken from the [International Aid Transparency Initiative](http://iatistandard.org/) (IATI) [Organisation Registration Agency Codelist](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) that provides an open, maintained list of reference codes for many Registration Agencies.
+In 360Giving data we ask publishers to use a list code taken from the [org-id list locator](http://org-id.guide/). This provides an open, maintained list of codes for many different lists around with world, giving a way to identify almost any organisation. 
 
-For example, a charity registered in England and Wales with the [Charity Commission](https://www.gov.uk/government/organisations/charity-commission) and a number of **1070468** will use an [IATI Organisation Registration Agency Code](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) of **GB-CHC**. The organisation identifier in your 360Giving data would be:
+```eval_rst
 
->GB-CHC-1070468
+.. admonition:: For example
 
-However, some organisations have more than one such identifier (e.g. where they are both a company and a charity), and others are not registered anywhere, and so you may only have names, or internal IDs, for them recorded on your data.
+  A charity registered in England and Wales with the Charity Commission of England and Wales, with the charity number '1070468' will use a prefix of ``GB-CHC``. 
 
-To give users of 360Giving data the best chance of joining up information about the same organisations across different data files, we follow a simple methodology to create Organisation Identifiers via the following process.
+  This gives an unique organisation identifier of ``GB-CHC-1070468``
 
-Work your way through the following steps. As soon as a step gives you an identifier, you can stop there and use the given identifier.
+```
 
-1. **Registered company number (UK)**
+### Choose the best identifier
 
-    If you have a registered company number for the organisation, use the Registration Agency Code 'GB-COH-' and the registered number (indicating that the number could be looked up at [Companies House](http://www.companieshouse.gov.uk))
+Some organisations have more than one identifier: they might be a charity **and** a company (charitable companies), or a charity **and** an educational establishment. 
 
-2. **Registered charity number (UK)**
+In these cases, it's important to know which identifier to pick so that users of data have the best possible chance of understanding that two grants have been made to the same organisation. 
```

Comment:
suggest updating for now:
"You may have more than one type of identifier for an organisation recorded in your system. In these cases, it will be necessary to pick which one to use when creating an Organisation identifier."

Reply:
Does that muddy it somewhat? How about 

> In these cases, it's important to pick an identifier that helps data users have the best possible chance of understanding that two grants have been made to the same organisation. 

?

Reply:
The follow two sentences (about org.id) and the 'hint' do some of the work of suggesting how to pick - and I was deliberately avoiding being more definite here, for now at least! I think this bit needs some implementation guidance because 'best possible chance' is context dependent in the 360 data.